src/app/dashboard-widgets/pipelines-widget/pipelines-widget.component.ts

```diff
@@ -39,12 +43,11 @@ export class PipelinesWidgetComponent implements OnInit {
 
   ngOnInit() {
     this.contextPath = this.context.current.map(context => context.path);
-    this.buildConfigs = this.pipelinesService.current.share();
+    this.buildConfigs = this.pipelinesService.getCurrentPipelines();
     // buildConfigsCount triggers changes in the DOM; force Angular Change Detection
     // via setTimeout encapsulation
     this.buildConfigs
       .map(buildConfigs => buildConfigs.length)
-      .share()
```

Comment:
Why don't these Observables need to be shared anymore?

Reply:
Right; I removed because they were in the wrong place but I forgot to actually add them to the right place. It should be correct now.